Multisig/Logic/Models/Safe.swift

```diff
@@ -118,16 +118,18 @@ extension Safe: Identifiable {
         } catch {
             throw GSError.DatabaseError(reason: error.localizedDescription)
         }
-
     }
 
-    static func create(address: String, name: String, selected: Bool = true) {
+    // Chain shouldn't be optional, this should be removed when completing the implementation
```

Comment:
how about `#warning` ?

Reply:
This is to not introduce warnings in our code base